pyproject.toml

```
@@ -1,13 +1,14 @@
 [build-system]
-requires = ["setuptools", "setuptools-scm"]
-build-backend = "setuptools.build_meta"
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
 
-[project]
-name = "SDS-in-a-box"
+[tool.poetry]
+name = "sds-data-manager"
 version = "0.1.0"
-description = "IMAP Science Operations Center SDS-in-a-box"
+description = "IMAP Science Operations Center AWS data manager"
+authors = ["Greg Lucas <greg.lucas@lasp.colorado.edu>, Bryan Harter <bryan.harter@lasp.colorado.edu>, Sean Hoyt <sean.hoyt@lasp.colorado.edu>, Tenzin Choedon <tenzin.choedon@lasp.colorado.edu>"]
```

```suggestion
authors = ["IMAP SDC Developers" <imap.sdc@lists.lasp.colorado.edu>]
```
Or add your name too :)

---

Forgot myself 🙃 I was going off the contributor list to the repo!